pkg/synthesis/synthesis.go

```diff
@@ -20,5 +29,5 @@ func NSXToK8sSynthesis(
 	allowOnlyPolicy := computeAllowOnlyRulesForPolicy(config.Fw.CategoriesSpecs, categoryToPolicy, allowOnlyFromCategory, hints)
 	abstractModel := &AbstractModelSyn{vms: parser.VMs(), epToGroups: parser.GetConfig().GroupsPerVM,
 		allowOnlyFromCategory: allowOnlyFromCategory, policy: []*symbolicPolicy{&allowOnlyPolicy}}
```

Comment:
these lines (28-31) better be encapsulated in a separate function, where it is clear that the input is the nsx config, and the output is the synthesis to abstract model.

Reply:
1. the input is also categoryToPolicy, allowOnlyFromCategory, hints, parser.VMs()
2. config is not exposed by the model pkg.

tried to encapsulate, does not look clearer